projectiles/NParticleBlast1/NParticleBlast1_proj.bp

```diff
@@ -36,6 +36,6 @@ ProjectileBlueprint {
         DestroyOnWater = false,
         InitialSpeed = 35,
         VelocityAlign = true,
-        UseGravity = false, -- should make unit not get stuck shooting AA
+        UseGravity = true, -- should make unit not get stuck shooting AA
```

Comment:
It get very small angle to projectile, so insted of stred line its more like uef projectiles (but smaller angle)